packages/styles/variables.css

```diff
@@ -7,6 +7,7 @@
 :root {
   /* color palette */
   --white: #ffffff;
+  --gray: #53636e;
```

Comment:
This isn't gray, if you look at the rgb values `83, 99, 110`, it's primarily blue with a bit of red and green. I don't think we should assign a generic variable name to something that isn't purely gray. We should name this something that will be unlikely to conflict with anything else.

Something like `--dark-accent`, we can always update it later.